src/auxiliary/containers.jl

```diff
@@ -50,8 +47,8 @@ length, the first `new_length` elements will be retained. If `new_length` is
 larger, the new elements are invalidated.
 """
 function Base.resize!(c::AbstractContainer, new_length)
-  @assert new_length >= zero(new_length) "New length must be >= 0"
-  @assert new_length <= capacity(c) "New length would exceed capacity"
+  @assert new_length>=zero(new_length) "New length must be >= 0"
+  @assert new_length<=capacity(c) "New length would exceed capacity"
```

Comment:
Maybe we should check whether it keep some whitespace around the operators if we add braces, e.g.,
```julia
  @assert (new_length >= zero(new_length)) "New length must be >= 0"
  @assert (new_length <= capacity(c)) "New length would exceed capacity"
```
What do you think?

Reply:
Does this only occur when used in macro arguments or also in other situations? I thought that binary operators are always to be surrounded by whitespace?

We can add parentheses to check if this restores the whitespace. However, TBH, the whole point of having automated formatting is to _not_ have to worry about formatting anymore. Thus I'd rather live with some questionable formatting choices (such as this) than having to remember this in the future.

We could also ask in the JuliaFormatter repo if this is fixable. However, if it only affects macros, I can live with it as well.